### PR TITLE
Draw X axis label inside the canvas and center Y axis label

### PR DESCRIPTION
**Pull request by bameda**

### CODE REVIEW

File: jquery.flot.axislabels.js [hunk 1]
```diff
@@ -100,10 +100,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
                         var x, y;
                         if (axisName.charAt(0) == 'x') {
                             x = plot.getPlotOffset().left + plot.width()/2 - width/2;
-                            y = plot.getCanvas().height;
+                            y = plot.getCanvas().height - 3;
```
Comment (yamila-moreno) [inline jquery.flot.axislabels.js:103]: random number?

File: jquery.flot.axislabels.js [hunk 2]
```diff
@@ -100,10 +100,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
...
                         } else {
                             x = height * 0.72;
-                            y = plot.getPlotOffset().top + plot.height()/2 - width/2;
+                            y = plot.getPlotOffset().top + plot.height()/2 + width/2;
```
Comment (yamila-moreno) [inline jquery.flot.axislabels.js:106]: and random math operation :+1: 

seems fine!

File: jquery.flot.axislabels.js (hunk 1 above)
Comment (bameda) [reply jquery.flot.axislabels.js:103]: Of course, a random number is as valid as any other :trollface:

File: jquery.flot.axislabels.js (hunk 2 above)
Comment (bameda) [reply jquery.flot.axislabels.js:106]: no, this one has sense: y = (the pointer at the top of the chart + (the height of the chart / 2) + (the width of the yaxis label / 2)

**Pull request merged**